<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogimagesubtracttool</class>
 <widget class="QMainWindow" name="dialogimagesubtracttool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Subtractor Tool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icon.qrc">
    <normaloff>:/icon/simpleProcessing.ico</normaloff>:/icon/simpleProcessing.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>410</width>
      <height>410</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(89, 89, 89);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_origin">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(89, 89, 89);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_compare">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>220</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(89, 89, 89);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>430</y>
      <width>201</width>
      <height>56</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Average: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Standard deviation:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLCDNumber" name="lcdNumber_avg">
         <property name="smallDecimalPoint">
          <bool>true</bool>
         </property>
         <property name="digitCount">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdnumber_SD">
         <property name="smallDecimalPoint">
          <bool>true</bool>
         </property>
         <property name="digitCount">
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>644</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionOpen_Image"/>
    <addaction name="actionImport_image_from_SimpleProcessing"/>
    <addaction name="actionSave_Image"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFiles"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Image">
   <property name="text">
    <string>Open image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionSave_Image">
   <property name="text">
    <string>Save image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionImport_image_from_SimpleProcessing">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Import image from SimpleProcessing</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>dialogimagesubtracttool</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
